python/ray/tune/logger.py

```diff
@@ -186,6 +186,8 @@ class TBXLogger(Logger):
         {"a": {"b": 1, "c": 2}} -> {"a/b": 1, "a/c": 2}
     """
 
+    VALID_HPARAMS = {str, bool, int, float, None}
```

Comment:
Should this include `list`?
```suggestion
    VALID_HPARAMS = {str, bool, int, float, list, None}
```

Reply:
Does list actually work?